policy/modules/services/cockpit.fc

```diff
@@ -0,0 +1,22 @@
+# cockpit stuff
+
+/etc/cockpit/ws-certs\.d(/.*)?			gen_context(system_u:object_r:cockpit_cert_t,s0)
+
+/run/cockpit(/.*)?						gen_context(system_u:object_r:cockpit_runtime_t,s0)
+/run/cockpit/certificate-helper(/.*)	gen_context(system_u:object_r:cockpit_cert_t,s0)
+
+/usr/lib/systemd/system/cockpit.*		--	gen_context(system_u:object_r:cockpit_unit_t,s0)
+
+/usr/libexec/cockpit-certificate-ensure	--	gen_context(system_u:object_r:cockpit_cert_manage_exec_t,s0)
+/usr/libexec/cockpit-ws		--	gen_context(system_u:object_r:cockpit_ws_exec_t,s0)
+/usr/libexec/cockpit-tls	--	gen_context(system_u:object_r:cockpit_ws_exec_t,s0)
+/usr/libexec/cockpit-wsinstance-factory	--	gen_context(system_u:object_r:cockpit_ws_exec_t,s0)
+/usr/libexec/cockpit-session	--	gen_context(system_u:object_r:cockpit_session_exec_t,s0)
+/usr/libexec/cockpit-ssh	--	gen_context(system_u:object_r:cockpit_session_exec_t,s0)
+
+/usr/share/cockpit/motd/update-motd    -- gen_context(system_u:object_r:cockpit_ws_exec_t,s0)
```

Comment:
Either because I don't think we wanted bin_t in cockpit.fc and/or because I didn't think we want /usr/share/cockpit/... in corecommands.fc (which upon further inspection might be OK) and/or because as I was working on this to get things running in enforcing I ended up with this and didn't give it much more thought.  

Either way, bin_t seems like a reasonable label.  I will do more testing to ensure it works as expected.